definitions/aiops-issue/definition.yml

```diff
@@ -0,0 +1,5 @@
+domain: AIOPS
+type: ISSUE
+configuration:
+  entityExpirationTime: MANUAL
+  alertable: true
```

Comment:
I just wondered if issues are alertable or is this used to display them in the explorer ?

Reply:
As I understand it's required so issues could be searched in the explorer

Reply:
If this will not have an alert status should be set as `alertable: false`.